internal/output/conform/template.go

```diff
@@ -0,0 +1,37 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package conform
+
+const configTemplate = `policies:
+- type: commit
+  spec:
+    dco: true
+    gpg: false
+    spellcheck:
+      locale: US
+    maximumOfOneCommit: true
+    header:
+      length: 89
+      imperative: true
+      case: lower
+      invalidLastCharacters: .
+    body:
+      required: true
+    conventional:
+      types: {{ .Types }}
+      scopes: {{ .Scopes }}
+- type: license
+  spec:
+    skipPaths:
+      - .git/
+    includeSuffixes:
+      - .go
+    excludeSuffixes:
+      - .pb.go
```

Comment:
We may need this configurable. I think there are a few projects that have something unique to them.

Reply:
yep, agreed, I went lazy route for now, but yeah, for sure. 

I was lazy to get full Go structs for it, so I ended up having something in between :)